cluster_tools/dockered-slurm/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: "2.2"
```

Comment:
I guess most files in `dockered-slurm` can be removed, right? If so, remove them and update the usages everywhere.

Reply:
I cleaned them up now, but the docker-compose.yml has to stay since it's used to spawn the slurm cluster.